ptrack/trackers.py

```diff
@@ -18,11 +18,14 @@ def __init__(self, name='ptrack'):
     def register(self, trackingpixel_callback):
         """ Register a new TrackingPixel """
         if inspect.isclass(trackingpixel_callback) is False:
-            raise Exception(trackingpixel_callback + "is not a class, must inherit from ptrack.TrackingPixel")
+            msg = trackingpixel_callback + "is not a class, must inherit from ptrack.TrackingPixel"
```

Comment:
Possibly out of scope but it feels like something other than Exception should be used here and below.